### PR TITLE
Sound updates

### PR DESCRIPTION
**Pull request by chaosvolt**
* Set a few subspells to be silent.
* Added sound messages for most attack spells.

Will be dependent on https://github.com/CleverRaven/Cataclysm-DDA/pull/34310

### CODE REVIEW

**Comment by chaosvolt** [comment]
Hmm. @korggent, out of curiosity, have you played around with how `mod_moves` behaves when the damage is positive? Once I get a chance to this this branch in a build I intend to see if the bonus moves I implemented in one spell work well from the player's POV, both when cast on oneself and when granting it to an NPC (I really hope that well-timed Surging Force used on an NPC will be basically "blink and the enemy they were attacking is sprawled out on the floor" but I suspect you'll still see them move and attack).

**Comment by KorGgenT** [comment]
i have not, but it should act as you expect: giving you extra moves.

File: Arcana/spells.json [hunk 1]
```diff
@@ -1,17 +1,30 @@
 [
+  {
+    "id": "arcana_aftermath_capacitance_stun",
+    "type": "SPELL",
+    "name": "Capacitance Stun",
+    "description": "This exists to make Capacitance sap moves from the user.",
+    "valid_targets": [ "self" ],
+    "flags": [ "SILENT" ],
+    "effect": "mod_moves",
+    "max_level": 10,
```
Comment (KorGgenT) [inline Arcana/spells.json]: is this spell meant to be cast alone? if not, i don't see a purpose for it having a max level, as the only value it has does not have an increment.

**Comment by KorGgenT** [comment]
short note: those other spell attributes related to sounds allow for sound effect hooks, though i'm not acquainted with the details.

**Comment by chaosvolt** [comment]
> i have not, but it should act as you expect: giving you extra moves.

Sounds good then. The main thing I was worried about was whether the effect in action, when it's not the player getting the bonus moves, would look too much like a freezeup or otherwise seem off to the player, but I'll see once I get the chance to test it.

> short note: those other spell attributes related to sounds allow for sound effect hooks, though i'm not acquainted with the details.

Ah right, I'm not very experienced with how sound hook properties work either, hence why I've only made rather limited usage of the feature so far.

File: Arcana/spells.json (hunk 1 above)
Comment (chaosvolt) [reply Arcana/spells.json]: Ah right, it's cast as a side effect of a spell that does level up. I'm guessing then that a sublevel doesn't need to have levels in this case?

Comment (KorGgenT) [reply Arcana/spells.json]: nope.

**Pull request merged**